### PR TITLE
Store host and claim info in sqlite as soon as possible

### PR DESCRIPTION
**Pull request by MrZammler**
##### Summary

In some cases, when the claim procedure happens shortly after the agent start, it might be possible that the `node_instance` and `host` tables are not yet populated. In this case, aclk does not find any nodes and as such does not send out any `CreateNodeInstance` messages.

The result is that the node does not appear as online ever in the cloud. This also affects our set team's test suite.

This PR reverts some recent changes, to populate these tables as soon as there are data for them.

##### Test Plan

Has been tested in set's test suite (but will be tested again).

If the agent is claimed a couple of seconds after start, it should appear online in the cloud.

##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
  
</details>

### CODE REVIEW

File: database/sqlite/sqlite_metadata.c [hunk 1]
```diff
@@ -1281,6 +1291,27 @@ void metaqueue_delete_dimension_uuid(uuid_t *uuid)
     queue_metadata_cmd(METADATA_DEL_DIMENSION, use_uuid, NULL);
 }
 
+void metaqueue_store_claim_id(uuid_t *host_uuid, uuid_t *claim_uuid)
+{
+    if (unlikely(!host_uuid))
+        return;
+
+    uuid_t *local_host_uuid = mallocz(sizeof(*host_uuid));
```
Comment (thiagoftsm) [inline database/sqlite/sqlite_metadata.c:1299]: I am reporting here an event that has relationship not only with this PR, but considering that is also happening here, I will  also share, please, take a look in the following report:

```sh
==28173== 152 bytes in 1 blocks are still reachable in loss record 3,278 of 4,108
==28173==    at 0x48475FF: calloc (vg_replace_malloc.c:1328)
==28173==    by 0x44A0A4: malloc_trace_find_or_create (libnetdata.c:219)
==28173==    by 0x44A4D5: mallocz_int (libnetdata.c:253)
==28173==    by 0x58E8B1: metaqueue_store_claim_id (sqlite_metadata.c:1299)
==28173==    by 0x66D043: load_claiming_state (claim.c:173)
==28173==    by 0x42B869: main (main.c:1957)
```

as you can see `queue_metdata_cmd` is not cleaning the allocated memory (ping @underhood ), probably we need to address this. @MrZammler do you want try a fix at least for this reported leak?

Comment (MrZammler) [reply database/sqlite/sqlite_metadata.c:1299]: Hi Thiago, thanks, yes, will check.

Comment (thiagoftsm) [reply database/sqlite/sqlite_metadata.c:1299]: After a check with @MrZammler we discovered that issue is appearing only when we use `NETDATA_TRACE_ALLOCATIONS` (ping @stelfrag ), considering that I do not expect users using this flag, I am going to approve the PR.

**Review by thiagoftsm — DISMISSED** [review]
PR is working as expected, LGTM!

**Review by stelfrag — APPROVED** [review]

**Review by stelfrag — CHANGES_REQUESTED** [review]
Unit test is failing

**Review by stelfrag — APPROVED** [review]

**Review by thiagoftsm — APPROVED** [review]
I retested PR, and I also ran unittest locally, LGTM!

**Comment by MrZammler** [comment]
Thanks guys, sorry for the mess...

**Pull request merged**